GitUI/CommandsDialogs/BrowseDialog/DashboardControl/UserRepositoriesList.cs

```diff
@@ -677,6 +681,32 @@ private void listView1_MouseLeave(object sender, EventArgs e)
             HoveredItem = null;
         }
 
+        private void listView1_GotFocus(object sender, EventArgs e)
+        {
+            if (listView1.Items.Count > 0 && listView1.SelectedItems.Count == 0)
+            {
+                listView1.Items[0].Selected = true;
+            }
+        }
+
+        private void listView1_KeyDown(object sender, KeyEventArgs e)
+        {
+            if (e.KeyCode == Keys.Up)
+            {
+                if (listView1.SelectedItems.Count > 0)
+                {
```

Comment:
Could you please merge these two and invert to reduce nesting?

Reply:
> p.s. I know that guard clauses is all the rage right now [youtube.com/shorts/Zmx0Ou5TNJs](https://www.youtube.com/shorts/Zmx0Ou5TNJs) (at least judging from my youtube recommendations) but I think context matters 😊

Now? Haha, more like in the past decade. Since people stopped following advice from a fellow named Uncle Bob (some of which I find rather questionable). There was a time when it was believed that a method must only have a single entry and a single return - .NET Framework code was written under those guidelines. It was rumoured that it yielded a better optimisation. Even if such anecdotes were true, these days it's immaterial, and code maintenability, readability and testability are significantly more important.